sdk/opa.go

```diff
@@ -241,6 +244,30 @@ func (opa *OPA) QueryDefault(ctx context.Context, input interface{}, opts ...fun
 	return opa.Query(ctx, *opa.manager.Config.DefaultDecision, input, opts...)
 }
 
+// AwaitReady waits for plugins and bundles to be loaded or until timeout is reached. Returns ready state.
+func (opa *OPA) AwaitReady(ctx context.Context, timeout time.Duration) (ready bool) {
```

Comment:
[nit] I'd rather not name the return,
```diff
- func (opa *OPA) AwaitReady(ctx context.Context, timeout time.Duration) (readya bool) {
+ func (opa *OPA) AwaitReady(ctx context.Context, timeout time.Duration) bool {
```

and use explicit `return true` instead of `ready = true` .... `return ready`

Reply:
Seems self-documenting to me, and as an added bonus editors like IntelliJ will be able to auto-complete this with appropriately named variables rather than just guesstimates. What's the downside?

Reply:
✔️ This function is short enough, I suppose. Coupled with naked returns, named return values can make it hard to grok code in larger functions.